starknet-contract/tests/contract_deployment.rs

```diff
@@ -64,6 +65,7 @@ async fn can_deploy_contract_to_alpha_sepolia_with_invoke_v1() {
 
 #[tokio::test]
 async fn can_deploy_contract_to_alpha_sepolia_with_invoke_v3() {
+    #![allow(clippy::or_fun_call)]
```

Comment:
Why can't we apply clippy suggestion here?

Reply:
Same as above.